core/modules/book/book.install

```diff
@@ -86,6 +86,9 @@ function book_schema() {
     ),
   );
 
+  // cache bin for book data
+  $schema['cache_book'] = backdrop_get_schema_unprocessed('system', 'cache');
+
```

Comment:
We usually specify a description for cache tables that is specific-use. e.g.https://api.backdropcms.org/api/backdrop/core!modules!filter!filter.install/function/filter_schema/1